tests/unit/test_plugin_adaptors.py

```diff
@@ -0,0 +1,97 @@
+from unittest import TestCase
+
+from pytest_splunk_addon_ui_smartx.utils import get_orca_deployment_urls
+
+
+def test_global_parameters(testdir):
+    testdir.makefile(".json", orca_deployment="""{
```

Comment:
you can also mock open function to avoid creating a file

Reply:
ty